docs/source/glm.rst

```diff
@@ -180,11 +184,8 @@ available link functions can be obtained by
    NegativeBinomial
    Power
    cauchy
-   cloglog
    identity
    inverse_power
    inverse_squared
-   log
-   logit
```

Comment:
(aside about the ancient part: lower case were instances at the beginning, but that messed up the code with mutable keyword arguments.)